src/selectors/events.js

```diff
@@ -199,15 +209,17 @@ export const selectFeaturedEventsByTitle = (state: State, title: string) => {
     return [];
   }
 
-  return ((featured.fields.events[locale].map(e =>
-    selectEventById(state, e.sys.id)
-  ): any): Event[]);
+  return featured.fields.events[locale].reduce((acc, item: Reference) => {
+    const event: ?Event = selectEventById(state, item.sys.id);
+    if (event) {
+      acc.push(event);
```

Comment:
Wouldn't `return acc.concat(event)` be preferrable?

Reply:
I would normally say yes, but because this reducer is in the same block as the initial, we can reuse the underlying array. Happy to change, just didn't see the reason to allocate more arrays.

Reply:
Makes sense, good to leave